backup.js

```diff
@@ -1,11 +1,12 @@
-var AWS = require('aws-sdk');
+var { S3Client } = require('@aws-sdk/client-s3');
+var { Upload } = require('@aws-sdk/lib-storage');
```

Comment:
I know you didnt introduce this pattern but we should be using `const` and not `var` for this